libs/designer/src/lib/ui/mcp/operations/ListOperations.tsx

```diff
@@ -70,6 +71,13 @@ export const ListOperations = () => {
     (operationId: string) => {
       dispatch(selectOperationIdToEdit(operationId));
       dispatch(openOperationPanelView());
+
+      LoggerService().log({
+        level: LogEntryLevel.Trace,
+        area: 'MCP.ListOperations',
```

Comment:
Edit operation is opening the edit operation panel, so lets call it MCP.EditOperation